uk.co.compucorp.civicrm.tasksassignments/views/contact/tasks.html

```diff
@@ -103,10 +108,23 @@ <h4 class="header-bar header-bar-danger">Resolved:
           <div class="{{prefix}}task-more">
             <div editable-ta="task.details" onbeforesave="updateTask(task, { details: ta.$data });">
               <div ta-bind ng-model="task.details"></div>
+              <i class="fa fa-pencil"></i>
             </div>
-            <p class="{{prefix}}task-contact-source">Created by:&nbsp; <a ng-href="{{url.CONTACT+'?reset=1&cid='+cache.contact.obj[task.source_contact_id].id}}"><span ng-bind="cache.contact.obj[task.source_contact_id].sort_name"></span></a></p>
           </div>
         </div>
+
+        <div class="col-xs-8">
+          <p class="{{prefix}}task-contact-source">
+            Created by:&nbsp;
```

Comment:
It's the paragraph  block in line 107 ^. It is currently inside the show more block. So to make consistent with the task list in T&A dashboard task list. I have moved to new row.